phoenix-core/src/test/java/org/apache/phoenix/schema/types/FileSystemBasedLobStoreTest.java

```diff
@@ -19,56 +19,38 @@
 package org.apache.phoenix.schema.types;
 
 import org.apache.commons.io.IOUtils;
-import org.junit.Test;
 
-import java.io.BufferedWriter;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.FileWriter;
+import org.junit.Test;
```

Comment:
Same here. This should be a separate class. Let's keep the fs-based stuff as is, as a simple LobStore implementation.

Reply:
+1